lib/pages/ui/nps_rating_bar.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+const String RATING_TYPE_NPS = "nps";
+
+class NpsRatingBar extends StatefulWidget {
+  final int counter;
+  final String minTitle;
+  final String maxTitle;
+
+  NpsRatingBar({Key key, this.counter, this.minTitle, this.maxTitle})
+      : super(key: key);
+
+  @override
+  _NpsRatingBarState createState() => _NpsRatingBarState(
+      counter: this.counter, minTitle: this.minTitle, maxTitle: this.maxTitle);
+}
+
+class _NpsRatingBarState extends State<NpsRatingBar> {
+  final int counter;
+  final String minTitle;
+  final String maxTitle;
```

Comment:
You can access these properties from `widget`, so that we don't need a constructor for them 🤔 
```dart
widget.counter
widget.minTitle
...
```

Reply:
thanks @markgravity 🙇  it has been a while that I forgot how to communicate from the state to the statefulWidget 😂